dolphinscheduler-ui/pnpm-lock.yaml

```diff
@@ -73,7 +57,7 @@ dependencies:
   echarts: 5.4.2
   js-cookie: 3.0.5
   lodash: 4.17.21
-  monaco-editor: 0.34.1
+  monaco-editor: 0.50.0
```

Comment:
Please revert this file, this change is not related with this PR.

Reply:
sure.